templates/devtest/gitea-ui.tmpl

```diff
@@ -253,5 +253,63 @@
 		{{template "shared/combomarkdowneditor" .}}
 	</div>
 	<script src="{{AssetUrlPrefix}}/js/devtest.js?v={{AssetVersion}}"></script>
+
+	<div>
+		<h1>Flex List</h1>
+		<div class="flex-list">
+			<div class="flex-card">
```

Comment:
For most frameworks, when there is a "list", then the "children elements" are called "item" 🤔 

"card" seems an uncommon concept and it conflicts with Fomantic UI card component.

Reply:
Agreed with @wxiaoguang

Reply:
Renamed `flex-card` to `flex-item`